R-packages/evalcast/vignettes/intro-evalcast.Rmd

```diff
@@ -44,87 +48,142 @@ library(evalcast)
 
 ```{r}
 err_measures <- list(wis = weighted_interval_score,
                      ae = absolute_error,
-                     coverage_80 = interval_coverage(alpha = 0.2))
+                     coverage_80 = interval_coverage(coverage = 0.8))
 ```
 
 We may now create scorecards for each forecaster's predictions or for a filtered subset. 
 
 ```{r}
-scorecards_yyg <- evaluate_predictions(filter_predictions(predictions_cards_yyg, ahead = 3), 
-                                   err_measures, 
-                                   backfill_buffer = 10)
-pcards_cmu_death3 <- filter_predictions(predictions_cards_cmu, 
-                                        ahead = 3,
-                                        response_data_source = "jhu-csse",
-                                        response_signal = "deaths_incidence_num")
-scorecards_cmu <- evaluate_predictions(pcards_cmu_death3, 
-                                   err_measures, 
-                                   backfill_buffer = 10)
-scorecards <- evaluate_predictions(filter_predictions(predictions_cards, ahead = 3), 
-                                   err_measures, 
-                                   backfill_buffer = 10)
-```
-
-Let's look at the `baseline_forecaster`'s scorecard for 3-epiweek ahead forecasts.
+scorecards <- evaluate_predictions(
+  bind_rows(predictions_cards, 
+            predictions_cards_cmu, 
+            predictions_cards_ens) %>%
+    # not always data for these locations
+    filter(! geo_value %in% c("us", "vi", "gu", "mp")),
+  err_measures = err_measures,
+  backfill_buffer = 10
+  )
+```
+
+
+`evaluate_predictions()` returns a long data frame with one column for each error measure added to the `predictions_cards`. Keeping all quantile predictions may be useful for plotting or other downstream analysis. For a more compact version, use `collapse_cards()` to return one row for each geo_value/ahead/forecast_date/forecaster combination.
 
 ```{r}
-scorecards[[1]]
+collapse_cards(scorecards) %>% str()
 ```
 
 **Technical note:** What does `backfill_buffer = 10` do?  When we evaluate a forecaster in backtesting we are assuming that we know what actually occurred.  However, in light of backfill, we may not trust the data for some period of time.  The argument `backfill_buffer` allows us to specify how many days until we believe that the data has "settled down" and is unlikely to be updated further.  The choice of this argument will depend on the particular signal you are forecasting.
 
 ### Plots of performance measures
 
-`evalcast` provides a number of tools to visually assess a forecaster (including comparisons with other forecasters):
+While `evalcast` provides a number of tools to visually assess a forecaster (including comparisons with other forecasters), because `score_cards` are just long data frames, many plots can be achieved using [{ggplot2}](http://ggplot2.tidyverse.org).
```

Comment:
Suggest moving the ggplot2 graph / comment below the evalcast ones. Basically, show off what evalcast can do, but then point out that you can customize as you like if evalcast isn't sufficient.